java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireExtensions.java

```diff
@@ -34,9 +34,23 @@ private SurefireExtensions() {
 
   public static List getExtensions() {
     return ImmutableList.of(
+        /**
+         * @deprecated
+         * @since 4.11
+         */
         PropertyDefinition.builder(SurefireUtils.SUREFIRE_REPORTS_PATH_PROPERTY)
             .name("JUnit Reports")
-            .description("Path to the directory containing all the *.xml JUnit report files. The path may be absolute or relative to the project base directory.")
+            .description("DEPRECATED - Path to the directory containing all the *.xml JUnit report files. The path may be absolute or relative to the project base directory.")
+            .onQualifiers(Qualifiers.PROJECT)
+            .category(JavaConstants.JAVA_CATEGORY)
+            .subCategory("JUnit")
+            .build(),
+        /**
+         * @since 4.11
+         */
+        PropertyDefinition.builder(SurefireUtils.SUREFIRE_REPORT_PATHS_PROPERTY)
+            .name("JUnit Report Paths")
+            .description("Comma-separated paths to the various directories containing the *.xml JUnit report files. Each path may be absolute or relative to the project base directory.")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)